src/parser/classes/VideoAttributeView.ts

```diff
@@ -0,0 +1,56 @@
+import { YTNode } from '../helpers.js';
+import { NavigationEndpoint } from '../nodes.js';
+
+import ContentPreviewImageView from './ContentPreviewImageView.js';
+import Parser from '../index.js';
+
+import type { RawNode } from '../types/index.js';
+
+export default class VideoAttributeView extends YTNode {
+  static type = 'VideoAttributeView';
+
+  image: ContentPreviewImageView | null;
+  image_style: string;
+  title: string;
+  subtitle: string;
+  secondary_subtitle: {
+    content: string
+  };
+  orientation: string;
+  sizing_rule: string;
+  overflow_menu_on_tap: {
+    innertube_command: NavigationEndpoint
+  };
+  overflow_menu_a11y_label: string;
+  logging_directives: {
```

Comment:
You can remove the logging_directives and it's children, generally YouTube.js avoids extracting the logging and tracking information, as nobody is really interested in it :)